src/main/java/org/humancellatlas/ingest/file/FileArchiveResult.java

```diff
@@ -0,0 +1,17 @@
+package org.humancellatlas.ingest.file;
+
+import java.time.Instant;
+import lombok.Data;
+
+@Data
+public class FileArchiveResult {
+    private Instant lastArchived;
+    private Boolean compressed;
```

Comment:
what does compressed mean?

Reply:
why is it needed?

Reply:
it indicates the file was compressed during archiving.

Reply:
is that info going to be passed to ENA?